README.rst

```diff
@@ -12,3 +12,101 @@ elife-crossref-xml-generation
 Crossref deposit of journal articles
 
 
+Overview
+========
+
+elife-crossref-xml-generation creates a Crossref content registration file for a journal article. The registration file is an XML file conforming to the Crossref schema. The code library is used by eLife in its publishing workflow to submit the content registration file by a POST request to the Crossref API endpoint, in order to register DOI values and metadata at Crossref. Following some basic changes to a configuration file, it should be able to generate Crossref content registration files for non-eLife journals.
+
+This library is intended for use by developers, typically integrated into a workflow. There are some options and decisions to make for how you connect it ot your existing workflows, or create an interface to convert individual JATS XML files via a web form, for example.
+
+Supported features
+==================
+
+* DOI
+* article title and basic metadata (volume, issue, pub date, issn, etc.)
+* contributors
+* abstract
+* Fundref
+* AccessIndicators
+* text and data mining (TDM) links
+* citation_list
+* components (if you have component DOI values)
+
+Ways to implement it
+====================
+
+The data passes through distinct steps, and at these steps the data can be altered if you would like to in your implementation of the library. The default data conversion begins with a JATS XML file, the XML is parsed to extract values into article objects, the article objects are used in a crossrefXML object to construct a Crossref XML representation of the data, and finally the CrossrefXML object can produce XML output. 
+
+  JATS XML -> article objects -> CrossrefXML object -> Crossref XML output
+
+There are situations you may want to manipulate the data prior to generating Crossref XML. For example, if the article XML is missing a publication date, or you want to specify a version of the article, then you could manipulate the objects in this way before you instantiate the CrossrefXML object.
+
+  JATS XML -> article objects -> manipulate the objects -> CrossrefXML object -> Crossref XML output
+
+Another way you can use this library
```

Comment:
not sure how the following `... -> ...` chain is different from the previous one?

Reply:
Without showing actual examples, I want to show how you can change the values of the objects during the stages of transformation. In the example I set the version of the article object before instantiating the CrossrefXML object. I wanted to draw some diagrams but I just had this shorthand text arrows to illustrate the point.